README.md

```diff
@@ -3,6 +3,8 @@
 This README would normally document whatever steps are necessary to get the
 application up and running.
 
+
```

Comment:
# Frontend re-review 1

- [ ] The mobile frontend implementation works, but its design is questionable.  It seems to span only half of the screen:

![image](https://github.com/otmanTR/nice-car-back-and/assets/57421823/583ad597-4f40-4a98-930f-c8395698b9e3)

![image](https://github.com/otmanTR/nice-car-back-and/assets/57421823/fc4d918d-f7e4-43f9-bdc1-55989adc153d)

![image](https://github.com/otmanTR/nice-car-back-and/assets/57421823/dceb8da1-1478-491b-8c66-28c4b9e6a7b7)

Kindly, apply styling so the phone screen can be filled with the app. 👍 


Tip:  Adding `height: 100vh` to the `.App` class seems to help:

![image](https://github.com/otmanTR/nice-car-back-and/assets/57421823/78d63271-069e-4def-9817-78a05676bba6)

From there, you could apply some more styling to center other elements.

- [ ] `AddCar.js` still has console.logs.  Kindly remove those debugging statements.

- [ ] OPTIONAL:  The dates and the form could use some validation.  I was able to make a reservation for a date in the past.  I was also able to set its ending date further in the past:

![image](https://github.com/otmanTR/nice-car-back-and/assets/57421823/d3540c8b-30d8-4b87-93de-3a004e9411bb)

- [ ] The frontend repo still needs a proper readme, as instructed in the previous review.  This is an important requirement for all projects.  Kindly do so in order to improve your chances in the final evaluation. 👍